integrations/jina/src/haystack_integrations/components/connectors/jina/reader.py

```diff
@@ -131,7 +131,7 @@ def run(self, query: str, headers: Optional[Dict[str, str]] = None):
         # raw response: we just return a single Document with text
         if not self.json_response:
             meta = {"content_type": response.headers["Content-Type"], "query": query}
-            return {"documents": [Document(content=response.content, meta=meta)]}
+            return {"documents": [Document(content=response.text, meta=meta)]}
```

Comment:
the `content` property is `bytes`. `text` is a `str`.
https://requests.readthedocs.io/en/latest/api/#requests.Response